src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/callback/AdjournCaseIt.java

```diff
@@ -44,474 +62,326 @@ public class AdjournCaseIt extends AbstractEventIt {
         setup();
-        setJsonAndReplace("callback/adjournCaseGeneratedVideoWhenCaseNotListedStraightAwayWithDirectionsMade.json", "DIRECTIONS_DUE_DATE_PLACEHOLDER", "2019-10-10");
+        setJsonAndReplace(NOT_LISTED_STRAIGHT_AWAY_WITH_DIRECTIONS_MADE_JSON, DIRECTIONS_DUE_DATE_PLACEHOLDER, DATE_2019);
 
-        MockHttpServletResponse response = getResponse(getRequestWithAuthHeader(json, "/ccdAboutToSubmit"));
-        assertHttpStatus(response, HttpStatus.OK);
-        PreSubmitCallbackResponse<SscsCaseData> result = deserialize(response.getContentAsString());
-
-        assertEquals(Collections.EMPTY_SET, result.getErrors());
-
-        assertNull(result.getData().getOutcome());
-
-        assertEquals(DRAFT_ADJOURNMENT_NOTICE.getValue(), result.getData().getSscsDocument().get(0).getValue().getDocumentType());
-        assertEquals(LocalDate.now().toString(), result.getData().getSscsDocument().get(0).getValue().getDocumentDateAdded());
-        assertEquals("Draft Adjournment Notice generated on " + LocalDate.now().format(DateTimeFormatter.ofPattern("dd-MM-YYYY")) + ".pdf", result.getData().getSscsDocument().get(0).getValue().getDocumentFileName());
+        noticeGeneratedWithExpectedDetails();
     }
 
-    /**
-     * Due to a CCD bug ( https://tools.hmcts.net/jira/browse/RDM-8200 ) we have had
-     * to implement a workaround in AdjournCaseAboutToSubmitHandler to set
-     * the generated date to now, even though it is already being determined by the
-     * preview document handler.  This is because on submission, the correct generated date
-     * (the one referenced in the preview document) is being overwritten to a null value.
-     * Once RDM-8200 is fixed and we remove the workaround, this test should be changed
-     * to assert that a "something has gone wrong" error is displayed instead of
-     * previewing the document, as a null generated date would indicate that the
-     * date in the preview document hasn't been set.
-     *
-     */
+    @DisplayName("Call to about to submit handler will write adjourn notice to case with generated date as set")
     @Test
-    public void callToAboutToSubmitHandler_willWriteAdjournNoticeToCaseWithGeneratedDateAsNowWhenGeneratedDateNotSet() throws Exception {
+    public void testDateSet() throws Exception {
         setup();
         setJsonAndReplace(
-            "callback/adjournCaseValidSubmissionWithNullGeneratedDate.json", "DIRECTIONS_DUE_DATE_PLACEHOLDER", "2019-10-10");
-
-        MockHttpServletResponse response = getResponse(getRequestWithAuthHeader(json, "/ccdAboutToSubmit"));
-        assertHttpStatus(response, HttpStatus.OK);
-        PreSubmitCallbackResponse<SscsCaseData> result = deserialize(response.getContentAsString());
-
-        assertEquals(Collections.EMPTY_SET, result.getErrors());
-
-        assertNull(result.getData().getOutcome());
+            "callback/adjournCaseValidSubmissionWithSetGeneratedDate.json", DIRECTIONS_DUE_DATE_PLACEHOLDER, DATE_2019);
```

Comment:
We could make this json path string a constant similar to NOT_LISTED_STRAIGHT_AWAY_WITH_DIRECTIONS_MADE_JSON. This also applies to the other cases within this class, might be worth going through each of them and adding constants.

Reply:
ok